src/plugins/Library/SpiderIndexURIs.java

```diff
@@ -82,29 +77,32 @@ synchronized FreenetURI loadSSKURIs() {
 			} finally {
 				Closer.close(fos);
 			}
-			try {
-				fis = new FileInputStream(new File(SpiderIndexUploader.EDITION_FILENAME));
-				BufferedReader br = new BufferedReader(new InputStreamReader(fis, "UTF-8"));
-				try {
-					edition = Long.parseLong(br.readLine());
-				} catch (NumberFormatException e) {
-					edition = 0;
-				}
-				Logger.debug(this, "Edition: "+edition);
-				fis.close();
-				fis = null;
-			} catch (IOException e) {
-				// Ignore
-				edition = 0;
-			} finally {
-				Closer.close(fis);
-			}
+//<<<<<<< HEAD
```

Comment:
Commented out code.

Reply:
Heh, that’s not only commented-out code, that’s a failed merge…